test/functions_topic_test.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+from src.functions_topic import *
```

Comment:
onboard `isort` as well, when you get around to that. And maybe `flake8`. Just take duckbot's config.

It's usually bad practice to do a `*` import, flake8 would yell at you for that.